File: 201223темтур/checker/gen.cpp
#include <iostream>
#include <vector>
#include <chrono>
#include <random>
using namespace std;

auto now = chrono::high_resolution_clock::now().time_since_epoch().count();
mt19937 rnd(now);

int randrange(int a, int b) {
    return a + rnd() % (b - a);
}

int main() {
    int n = randrange(17, 20);
    vector<int> a = {0};
    
}

File: тренировки к перечням/вузакотбор 2024/C/gen.py
from random import *

n, q = randrange(40, 100), randrange(1, 3)

print(n, q)
for i in range(q):
    x = randrange(1, n + 1)
    d = randrange(0, n - 1 + 1)
    print(x, d)